### DISCUSSION TITLE
Can typesense be moved to other hosts for processing when importing a large number of photos for the first time?

### DISCUSSION THREAD

**Discussion by davidpan** [immich-app/immich#5208] [General]
Currently, when importing a large number of photos for the first time, typesense requires more memory and disk speed to process face recognition merge. Is it feasible to transfer it to other machines like ML?

**Comment by jrasm91** [comment]
Yeah there are some env variables for considering the typesense host. See the docs:

https://immich.app/docs/install/environment-variables

**Reply by davidpan** [reply]
thanks.
